### PR TITLE
[CORE-2959] Update carrier reference data (Deu Post and Asendia)

### PR DESCRIPTION
**Pull request by bhavya-verma-flow**
JIRA ticket: https://flowio.atlassian.net/browse/CORE-2959
This PR contains the addition of a new carrier. After the approval, I'm gonna add the new service levels in this PR only.

### CODE REVIEW

**Review by paololim — CHANGES_REQUESTED** [review]
We also need to add the services (see carriers-services.csv) and then run `reference.go` to make sure all the JSON reference data is properly generated for carriers and carrier services.

**Comment by paololim** [comment]
This comment contains the list of everything you need to add: https://flowio.atlassian.net/browse/CORE-2959?focusedCommentId=36249

File: common/common.go [hunk 1]
```diff
@@ -374,3 +381,27 @@ func RemapCurrencyCodeToSupported(code string) string {
 		return newCurrency
 	}
 }
+
+func MoveFile(sourcePath, destPath string) error {
```
Comment (bhavya-verma-flow) [inline common/common.go:385]: Reference for this method: https://gist.github.com/var23rav/23ae5d0d4d830aff886c3c970b8f6c6b

Comment (mikeclimbrock) [reply common/common.go:385]: mind leaving this as an actual comment in the code for future reference?

**Review by mikeclimbrock — APPROVED** [review]
lgtm!

**Comment by bhavya-verma-flow** [comment]
@mikeclimbrock You are talking about the below comment to be added into the code
https://github.com/flowcommerce/json-reference/pull/110#discussion_r686020944
Also do I need to run reference.go file and then only merge this PR as the JSON reference data is only gonna be generated after running that file.

File: data/original/carriers.csv [hunk 2]
```diff
@@ -2,6 +2,7 @@ id,name,tracking_url
 asendia,Asendia,https://track.aftership.com/asendia/
 canada-post,Canada Post,https://track.aftership.com/canada-post/
 chronopost,Chronopost,https://www.chronopost.fr/en/chrono_suivi_search?listeNumerosLT=
+deutsche-post,Deutsche Post,https://track.aftership.com/deutsch-post/
```
Comment (paololim) [inline data/original/carriers.csv:5]: is there a Deutsche Post URL for tracking?

Comment (bhavya-verma-flow) [reply data/original/carriers.csv:5]: https://www.deutschepost.de/sendung/simpleQuery.html?locale=en_GB
This is the one I found out for Deutsche Post but not sure if it is correct.

**Review by paololim — APPROVED** [review]
thanks!

after merging this PR, you will need to run `go run release.go` on https://github.com/flowcommerce/lib-reference-scala

**Comment by bhavya-verma-flow** [comment]
Ack

**Pull request merged**